en/snap/platform/overview.md

```diff
@@ -399,26 +399,24 @@ Here is an example of 1 order that has 1 earlier payment id which is unpaid, and
 ##### Payment Method Icons on Shopify Payment Setting Page
 On the Shopify payment setting page, you will find there are payment method icons. But be advised that **the icons selection does not directly affect the actual payment methods activated on Midtrans**.
 ![Payment icons](./../../../asset/image/shopify-new-25-payment-icons.png ':size=400')<br>
-The selected icons will be shown on the payment button on the Shopify checkout page. It does not affect the actual payment methods on Midtrans' Snap payment page.
+The selected icons only will be shown on the payment button on the Shopify checkout page.
 ![Payment icons](./../../../asset/image/shopify-new-25-payment-icons-on-checkout.png ':size=400')<br>
 If you want to configure active payment methods on the Snap payment page, you should configure it on [Midtrans Shopify Payment App Account Settings](#payment-app-account-settings).
 
 ##### Shopify will not Charge Third-Party Transaction Fees for Test Order with Enabling Test Mode
-"Test order" is helpful for you to make sure that the checkout process and settings for order processing, payment, inventory, shipping, email notifications, etc are all correct. You can place a test order by [simulating a transaction through test mode](#test-transaction-with-sandbox-mode).
+In real transaction, when you use a third-party payment provider such as "Midtrans", Shopify will add transaction fee for each transaction, the transaction fee varies depending on your [pricing plan](https://www.shopify.com/id/pricing).
 
-When you use a third-party payment provider like "Midtrans" to take customer payments, shopify will add transaction fee for each transaction, the transaction fee varies depending on your [pricing plan](https://www.shopify.com/id/pricing).
-
-But if you want to do "test order" with enabling test mode, **Shopify should not charge you any payment processing fee for "test order" payments.**, all orders made within this mode are not "real", and do not require "real payment/fund", so you are **safe to ignore the payment for this order** because the payment is not real and the order should be safe to not actually fulfilled, you can check [shopify docs about test order](https://help.shopify.com/en/manual/checkout-settings/test-orders) for the detail.
+But if you want to do "Test Order" with enabling test mode, **Shopify should not charge you any payment processing fee for "Test Order" payments.** "Test Order" is helpful for you to make sure that the checkout process and settings for order processing, payment, inventory, shipping, email notifications, etc are all correct. You can place a "Test Order" by [simulating a transaction through test mode](#test-transaction-with-sandbox-mode). You can also check [Shopify docs about test order](https://help.shopify.com/en/manual/checkout-settings/test-orders) for further details.
 
```

Comment:
> with enabling test mode

bad

> "Test Order" is helpful for

bad. Do you have difficulty in [reading this previous review](https://github.com/Midtrans/technical-documentation-site/pull/95#discussion_r974887417)?
- Performing "Test Order" ...

>  by [simulating a transaction through test mode](#test-transaction-with-sandbox-mode)

Redundant. improper capitalization. should be
- by transacting using Test Mode

> Shopify docs about test order

inconsistent